packages/dev/core/src/Physics/v2/Plugins/havokPlugin.ts

```diff
@@ -52,18 +53,53 @@ class MeshAccumulator {
      * have a physics impostor. This is useful for creating a physics engine
      * that accurately reflects the mesh and its children.
      */
-    public addMesh(mesh: Mesh, includeChildren: boolean): void {
-        const indexOffset = this._vertices.length;
-        // Force absoluteScaling to be computed
+    public addNodeMeshes(mesh: TransformNode, includeChildren: boolean): void {

```

Comment:
This is probably a breaking change. You can keep the old function and proxy it to the new version while deprecating it (using the `@deprecated` tag in the tsdoc comment).

Reply:
Class is used internally, no breaking change.